### DISCUSSION TITLE
每个单独服务如何配置单独的 数据库配置或者redis配置？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#252] [02.提问] [answered]
每个单独服务如何配置单独的 数据库配置或者redis配置？
比如我a服务需要连接mysql db1,b服务需要连接mysql db2，a服务需要连接redis database0 ,b服务需要连接 redis database 1？
这块nacos那边需要怎么配置？ 有现成的实现吗 还是需要自己去二次开发

**Comment by zuihou** [comment, marked as answer]
mysql.yml. redis.yml 等是全局配置文件，所有服务都会读。
lamp-xxx-server.yml、lamp-yyy-server.yml 等是各个服务自己的配置文件。

读取顺序为： lamp-xxx-server.yml > mysql.yml.


所以 xxx 服务要访问A db， 只需要将数据库配置复制到  lamp-xxx-server.yml  中，即可覆盖mysql.yml 中的全局配置。


reids 同理

**Reply by zqb666** [reply]
谢谢
